docs/sources/configuration/_index.md

```diff
@@ -2499,14 +2499,14 @@ This way, one doesn't have to replicate configuration in multiple places.
 [ring: <ring>]
 ```
 
-## usage_report
+## analytics
 
-This block allow to configure usage report of Loki to grafana.com
+This block allows to configure analytics reporting of Loki to grafana.com
```

Comment:
For future writing tasks, the sentence construction "allows to configure" (allows to something) is not proper English grammar. Instead of this, leave out the "allows to" phrase, and use the present tense of the verb ("configure" in this case).
 
```suggestion
The `analytics` block configures the reporting of Loki analytics to grafana.com
```